packages/react/src/components/DatePicker/DatePicker.test.tsx

```diff
@@ -1,21 +1,28 @@
+import '@testing-library/jest-dom';
+
 import * as React from 'react';
-import { Calendar } from '../../Calendar';
 import { DatePicker } from './DatePicker';
+import { Calendar } from '../Calendar';
 import { DatePickerBase } from './DatePicker.base';
 import { FirstWeekOfYear } from '@fluentui/date-time-utilities';
-import { shallow, mount, ReactWrapper } from 'enzyme';
-import { resetIds } from '@fluentui/utilities';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { resetIds, setWarningCallback } from '@fluentui/utilities';
 import { Callout } from '../../Callout';
 import { safeCreate } from '@fluentui/test-utilities';
 import { TextField } from '../../TextField';
 import * as renderer from 'react-test-renderer';
 import * as ReactDOM from 'react-dom';
 import { CalendarDayGridBase } from '../CalendarDayGrid/CalendarDayGrid.base';
 import { isConformant } from '../../common/isConformant';
-import type { IDatePickerStrings, IDatePickerProps } from './DatePicker.types';
+import { IDatePickerStrings } from './DatePicker.types';
 
 describe('DatePicker', () => {
   beforeEach(() => {
+    setWarningCallback(() => {
```

Comment:
why is this needed ? did the refactor started to trigger some deprecation warnings ?